spec/board_spec.rb

```diff
@@ -8,4 +8,23 @@
       expect(board.board).to eq([1, 2, 3, 'x', 5, 6, 7, 8, 9])
     end
   end
+
+  describe 'tracking remaining moves' do
+    it 'when a move is made, the number of moves remaining decrements by 1' do
```

Comment:
Try to use the `it` to your advantage where possible. For example, here it could be:
`it 'decreases the remaining moves by one`

Reply:
Thanks for spotting this, Matt. I'll ensure the description of the tests take into consideration `it`